samples/batch/transcription-enabled-storage/CreateByTimerFunction/Properties/ServiceDependencies/CreateTranscriptionFunction-a-jav-test - Zip Deploy/profile.arm.json

```diff
@@ -0,0 +1,176 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_dependencyType": "function.windows.premium"
+  },
+  "parameters": {
+    "resourceGroupName": {
+      "type": "string",
+      "defaultValue": "a-jav-test",
+      "metadata": {
+        "description": "Name of the resource group for the resource. It is recommended to put resources under same resource group for better tracking."
```

Comment:
This is not a really meaningful description. :-)

Reply:
True that. Especially since this is an auto-generated file that should have never ended up here. Thanks for noticing!